.env-example

```
@@ -1,3 +1,6 @@
+ENTERPRISE_ENABLED=false
```

There seems to be a mismatch between this name and the one inside of `config/secrets.yml`:

```suggestion
GITHUB_ENTERPRISE_ENABLED=false
```

---

Instead, I think this might be better just detecting if a GITHUB_ENTERPRISE_HOSTNAME is present.